drivers/clk/msm/mdss/mdss-pll-util.c

```diff
@@ -260,6 +260,9 @@ static int mdss_pll_util_parse_dt_supply(struct platform_device *pdev,
 
 		mp->vreg_config[i].post_off_sleep = (!rc ? tmp : 0);
 
+		mp->vreg_config[i].drms_unsupported = of_property_read_bool(supply_node,
```

Comment:
if regulator rdesc.name contains "gdsc", then drms_unsupported = true;

That's the right way of doing it.

Reply:
P.S.: The regulator name is always "gdsc_xxxx"